index.html

```diff
@@ -0,0 +1,257 @@
+						</div>
+						<div id="tab3" class="tab-content tab-content--3">
+							<h2 class="tab-content__title tab-content__title--3">Text from box name 1</h2>
+							<p class="tab-content__text">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis ornare turpis nec mauris scelerisque, sed suscipit elit dapibus. Aliquam ut eros sed risus convallis venenatis eget vitae mi. Cras dictum est non nunc semper blandit. Quisque convallis tristique dolor eget ultrices. Mauris ac orci quis neque facilisis laoreet. Suspendisse vitae ultrices ipsum, at placerat eros. Proin varius malesuada ultricies. Cras a metus elementum, pellentesque libero eu, faucibus lorem. Integer ac ornare lacus. Ut pretium felis tellus, nec faucibus eros rhoncus eget. Pellentesque ultrices, ligula vestibulum porttitor pharetra, arcu nisl vulputate ligula, id vulputate ligula urna vel tortor. In ullamcorper blandit erat, nec egestas massa commodo consectetur. In nec odio lorem. Donec nulla neque, vehicula in bibendum et, rhoncus sit amet nibh.</p>
+						</div>
+						<div id="tab4" class="tab-content tab-content--4">
+							<h2 class="tab-content__title tab-content__title--4">Text from box name 1</h2>
+							<p class="tab-content__text">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis ornare turpis nec mauris scelerisque, sed suscipit elit dapibus. Aliquam ut eros sed risus convallis venenatis eget vitae mi. Cras dictum est non nunc semper blandit. Quisque convallis tristique dolor eget ultrices. Mauris ac orci quis neque facilisis laoreet. Suspendisse vitae ultrices ipsum, at placerat eros. Proin varius malesuada ultricies. Cras a metus elementum, pellentesque libero eu, faucibus lorem. Integer ac ornare lacus. Ut pretium felis tellus, nec faucibus eros rhoncus eget. Pellentesque ultrices, ligula vestibulum porttitor pharetra, arcu nisl vulputate ligula, id vulputate ligula urna vel tortor. In ullamcorper blandit erat, nec egestas massa commodo consectetur. In nec odio lorem. Donec nulla neque, vehicula in bibendum et, rhoncus sit amet nibh.</p>
+						</div>
+					</div>
+				</div>
+			</div>
+		</section>
+	</main>
+	<!-- end of main -->
+
+	<!-- footer -->
+	<footer class="footer">
```

Comment:
Tak w ogole to odstępy - indents - są za duże. Źle się czyta ten kod. Powinnaś je zmniejszyć. Nie wiem jakiego IDE używasz ale powinnaś mieć tam opcję aby to ustawić.